client/my-sites/theme/main.jsx

```diff
@@ -885,20 +885,31 @@ class ThemeSheet extends Component {
 		}
 
 		if ( hasWpOrgThemeUpsellBanner || hasThemeUpsellBannerAtomic ) {
+			const thisPageUrl = `/theme/${ themeId }/${ siteSlug }`;
 			pageUpsellBanner = (
 				<UpsellNudge
 					plan={ PLAN_BUSINESS }
 					className="theme__page-upsell-banner"
-					title={ translate( 'Access this theme for FREE with a Business plan!' ) }
+					title={
+						siteId
+							? translate( 'Upgrade your plan to access third-party themes' )
```

Comment:
Nitpick: What if we also mention the Business plan in the nudge title?
```suggestion
							? translate( 'Upgrade to a Business plan to access third-party themes' )
```

Reply:
The repeated use of 'to' is valid English but it doesn't feel as slick for an upgrade nudge. I used some different wording instead that explicitly mentions the business plan and is still clear about this being a third party theme